.github/workflows/test-package.yml

```diff
@@ -77,3 +77,31 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           path-to-lcov: coverage/lcov.info
+
+  benchmark:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
In general, we've been pushing towards pinning actions at the latest version released version with a SHA – FYI